google-cloud-gaming/Gemfile

```diff
@@ -2,4 +2,4 @@ source "https://rubygems.org"
 
 gemspec
 
-gem "google-cloud-gaming-v1", path: "../google-cloud-gaming-v1"
+gem "google-cloud-gaming-v1", path: "../obsolete/google-cloud-gaming-v1"
```

Comment:
Note that this is the only manual change in order to avoid breaking the tests. 

PR#23635 will have this file removed from the non-versioned gem (to be merged after).